app/views/layout.njk

```diff
@@ -46,11 +51,29 @@
 
   {# Place holder section for things like back links or breadcrumbs #}
   {% block breadcrumbs %}
+    {#  The back link should provide an object like this: { text: 'Back', herf: '/link/to/previous/page'  #}
+    {% if backLink %}
+      {{ govukBackLink(backLink) }}
+    {% endif %}
   {% endblock %}
 {% endblock %}
 
 {% block content %}
-  <h1 class="govuk-heading-xl">Default page template</h1>
+  {# Pages can provide an 'errorList' to show the error summary. It should look like this:
+    [{ text: 'An error', href: "#link-to-form-item"}]  #}
+  {% if errorList %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errorList
+    }) }}
+  {% endif %}
+
+  {# We set the page heading as a variable in the view to ensure all pages are consistent use this in your form field
+    when setting 'isPageHeading: true' #}
```

Comment:
it's the other way around. if you don't want/need to use the default one from the macro then set `isPageHeading: true` and it will create the h1 otherwise leave it off and it will create a label

https://github.com/alphagov/govuk-frontend/blob/main/packages/govuk-frontend/src/govuk/components/label/template.njk#L12